### ISSUE TITLE
EMUI don't work

### ISSUE THREAD

**Issue by bobaoapae** [Magisk-Modules-Repo/quickstepswitcher#2]
In EMUI 9, instead of defining this value inside some xml resource, Huawei decided to create a jar file instead named "hwServices.jar" & hardcoded EMUI launcher's package name in a "String" variable to be used for triggering the QUICKSTEP intent. 

Check: https://forum.xda-developers.com/huawei-p20-pro/themes/root-emui-9-qwickstep-recents-screen-t3914487

**Comment by skittles9823** [comment]
Yes, I'm aware of this and I've been talking with Aod on Telegram about this issue. 

Our internal test group for QuickSwitch has been brainstorming how to get around this but it may take some time if we can figure out anything that will work at all.
